src/scss/foundations/typography/index.scss

```diff
@@ -117,6 +119,13 @@ $_letterSpacing: (
   font-weight: #{lib.getOrDie($_font, weights, $weight)};
 }
 
+/// Include the font-weight strong
+///
+/// @access public
+@mixin getStrongWeight() {
+  @include _fontWeight(bold);
+}
```

Comment:
Non mi fa impazzire come soluzione per vari motivi: 
- è un praticamente un alias di una funzione che già esiste ma che non stiamo esponendo. Quindi forse dobbiamo valutare se è giusto che sia privata 
- esiste solo il mixin per il bold e non per gli altri pesi 

In generale piuttosto che avere questo mixin farei una cosa del tipo: 
```scss
strong {
  @include typography.text(small, bold);
}
```
In questo modo si utilizza il mixin pubblico come è giusto che sia e inoltre non è affatto sbagliato dire che i tag puri, se non specificato diversamente attraverso classi o atoms, hanno una loro font-size di default.